crates/p2p/src/lib.rs

```diff
@@ -84,6 +84,13 @@ pub struct Config {
     pub ip_whitelist: Vec<IpNet>,
     pub bootstrap: BootstrapConfig,
     pub inbound_connections_rate_limit: RateLimit,
+    /// Alternative protocol names for Kademlia
+    pub kad_names: Vec<String>,
+    /// Request timeout for p2p-stream
+    /// TODO change the semantics to timeout since last response
```

Comment:
I think we should convert this TODO into an issue instead? It's unlikely we'll check this TODO here.

Reply:
We have an issue for this https://github.com/eqlabs/pathfinder/issues/2120